vignettes/advanced.Rmd

```diff
@@ -222,15 +222,15 @@ x <- y1 %>%
     version = issue,
     percent_cli = value
   ) %>%
-  as_epi_archive()
+  as_epi_archive(compactify=FALSE)
```

Comment:
For vignettes, we should use `compactify=TRUE` if possible.  I used `compactify=FALSE` in the data generation because otherwise some archive-related tests that relied on the uncompactified version of the example archive would fail.